react/index.tsx

```diff
@@ -110,27 +70,17 @@ const OrderPlaced: FC = () => {
           <title>{intl.formatMessage(messages.title)}</title>
         </Helmet>
 
-        <article className="pt9 sans-serif">
+        <div className={`${handles.orderPlacedWrapper} pt9 sans-serif`}>
           {/* <Analytics eventList={orderGroup.analyticsData} /> */}
```

Comment:
Useful?

Reply:
Was already there, just commented it out for testing purposes. Will uncomment in the future